aiokafka/conn.py

```diff
@@ -443,7 +441,7 @@ def send(self, request, expect_response=True):
     def _send_sasl_token(self, payload, expect_response=True):
         if self._writer is None:
             raise Errors.KafkaConnectionError(
-                "No connection to broker at {0}:{1}"
+                "No connection to broker at {}:{}"
                 .format(self._host, self._port))
```

Comment:
?

Reply:
I've missed this, and some other `format` occurrences. Fixed in c7e78f19bf2dea2b3a10fd82273cc3b0f7144e5c